jekyll/_cci2/faq.adoc

```diff
@@ -0,0 +1,83 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+  - Server v2.x
+---
+= FAQ
+:categories: ["migration"]
+:description: Frequently asked questions about CircleCI
+:experimental:
+:page-layout: classic-docs
+:short-title: FAQ
+
+This page answers frequently asked questions about CircleCI.
+
+[#general]
+== General
```

Comment:
I wonder if it would be good to add the main h2 sections in a list at the top of the page to help navigate as it is so long?